libkbfs/tlf_journal.go

```diff
@@ -346,6 +379,13 @@ const (
 // enqueued journal ID tag.
 const CtxJournalOpID = "JID"
 
+func (j *tlfJournal) pause(pauseType tlfJournalPauseType) {
```

Comment:
My first attempt at this left `j.wg.Pause` where it was, but unfortunately there's a race if you leave it there.  It's possible for CR to kick off and resolve the branch before the pause is processed by the background goroutine.  That leads to `Resume` being called on an unpaused wg, which leads to a panic.  I think calling `j.wg.Pause` right at branch conversion is the only way to solve that problem, so I don't think I can take this suggestion.